javagen/src/main/java/com/azure/autorest/util/SchemaUtil.java

```diff
@@ -254,12 +254,13 @@ public static Optional<ClassType> mapExternalModel(ObjectSchema compositeType) {
                         && compositeType.getLanguage().getJava().getNamespace() != null) {
 
                     // https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/models/PollResult.java
-                    if (Objects.equals(name, "PollResult")
-                            && compositeType.getLanguage().getJava().getNamespace().startsWith("com.azure.core")) {
+                    if ("PollResult".equals(name)
+                        && compositeType.getLanguage().getJava().getNamespace().startsWith("com.azure.core")) {
                         classType = new ClassType.Builder()
-                                .name(name)
-                                .packageName(compositeType.getLanguage().getJava().getNamespace())
-                                .build();
+                            .name(name)
+                            .packageName(compositeType.getLanguage().getJava().getNamespace())
+                            .usedInXml(treatAsXml(compositeType))
```

Comment:
We may not need this line. This [`PollResult`](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/models/PollResult.java) is only used for LRO poll result. I guess it won't be XML?  (emm, do we have LRO with XML?)

Reply:
I don't think we have an XML-based `PollResult` so this could be removed.